scripts/linters/python_linter.py

```diff
@@ -317,6 +338,30 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    def _get_trimmed_error_messages(self, lint_messages):
+        """Remove extra bits from pylint messages.
+
+        Args:
+            lint_messages: list. Messages returned by the python linter.
+
+        Returns:
+            str. A string with the trimmed messages.
+        """
+        error_messages = ''
+        # Remove newlines and coverage report from the end of message.
+        lint_messages = lint_messages[:-5]
+        for message in lint_messages:
+            python_utils.PRINT(message)
+            # Every pylint message has a message id inside the brackets
+            # so we need to check if it is true and if case is true then
+            # remove the message-id from the end of original message.
```

Comment:
Answered this above.

Reply:
Can you please add the link to the comment as I cannot find one?

Reply:
I don't know where that comment go. So I am explaining again. We are printing messages here as well for the same reason if someone want to know what is the message-id they can scroll up. And I removed the message id because Sean doesn't want that in the ouput.

Reply:
When you say "Sean doesn't want that in the output", could you please add a reference? (Do this any time you quote me -- or anyone else.)